QuestPDF/Drawing/FontManager.cs

```diff
@@ -100,5 +105,13 @@ internal static SKFontMetrics ToFontMetrics(this TextStyle style)
         {
             return FontMetrics.GetOrAdd(style.FontMetricsKey, key => style.ToPaint().FontMetrics);
         }
+
+        private static float GetTextScale(TextStyle style)
+        {
+            if (style.FontPosition == FontPosition.Superscript || style.FontPosition == FontPosition.Subscript)
+                return 0.65f;
```

Comment:
> What is the origin of this value? I am looking at popular configurations [here](https://en.wikipedia.org/wiki/Subscript_and_superscript#Software_support).

Taken from [here](https://github.com/toptensoftware/RichTextKit/blob/02611ab62e62986e7dbcb4528c0ea965bef62b96/Topten.RichTextKit/TextShaping/TextShaper.cs#L339). But we can definitly use other values, e.g `0.58` which seems to be the most common one in the source you referenced.